test/utils/content-encoding.spec.ts

```diff
@@ -39,19 +39,19 @@ describe("ContentEncoding", () => {
   describe("#supportedAlgorithm", () => {
     it("returns true when content-encoding is a supported algorithm", () => {
       expect(contentEncoding.supportedAlgorithm()).to.eql(true)
+      setEncoding("br")
+      expect(contentEncoding.supportedAlgorithm()).to.eql(true)
     })
 
-    it("returns true when content-encoding is not a supported algorithm", () => {
-      setEncoding("br")
-      expect(contentEncoding.supportedAlgorithm()).to.eql(false)
+    it("returns false when content-encoding is not a supported algorithm", () => {
       setEncoding("identity")
       expect(contentEncoding.supportedAlgorithm()).to.eql(false)
     })
   })
 
   describe("#uncompressedBody", () => {
```

Comment:
Mind adding a similar test to the gzip one that tests actual compressed content.
The same for `#compressBody`.

Reply:
Sure thing! Updated to include the tests.